src/styles/blocks/bikes.scss

```diff
@@ -0,0 +1,94 @@
+.bikes {
+
+  &__content {
+
+    @include onTablet {
+      margin-top: 132px;
+    }
+
+    @include onDesktop {
+      margin-top: 190px;
+    }
+  }
+
+  &__catalog {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__card {
+    margin-bottom: 40px;
+    width: 100%;
+
```

Comment:
checklist 8: Pictures in Gallery should increase on hover

Reply:
thx, i fixed it